--- header.tpl	Fri Apr 29 17:17:54 2011
+++ header_patched.tpl	Fri Apr 29 17:24:12 2011
@@ -6,6 +6,26 @@
 
 <div id="headerContainer">
 	<a id="top"></a>
+	{if $this->getUser()->userID != 0}
+	<script type="text/javascript">
+	//<![CDATA[
+	function log_out()
+	{
+		grayfilter = document.getElementsByTagName("html");
+		grayfilter[0].style.filter = "progid:DXImageTransform.Microsoft.BasicImage(grayscale=1)";
+		if (confirm('{lang}wbb.sebijk.logout.main{/lang}\n {lang}wbb.sebijk.logout.yesno{/lang}'))
+		{ 
+		return true; 
+		}
+		else
+		{
+		grayfilter[0].style.filter = "";
+		return false; 
+		}
+	}
+	//]]>
+	</script>
+	{/if}
 	<div id="userPanel" class="userPanel">
 		<div class="userPanelInner">
 			<p style="display: none;" id="userAvatar">
@@ -17,7 +37,7 @@
 			<div id="userMenu">
 				<ul>
 					{if $this->user->userID != 0}
-						<li id="userMenuLogout"><a href="index.php?action=UserLogout&amp;t={@SECURITY_TOKEN}{@SID_ARG_2ND}"><img src="{icon}logoutS.png{/icon}" alt="" /> <span>{lang}wbb.header.userMenu.logout{/lang}</span></a></li>
+						<li id="userMenuLogout"><a href="index.php?action=UserLogout&amp;t={@SECURITY_TOKEN}{@SID_ARG_2ND}" onclick="return log_out()"><img src="{icon}logoutS.png{/icon}" alt="" /> <span>{lang}wbb.header.userMenu.logout{/lang}</span></a></li>
 						<li id="userMenuProfileEdit"><a href="index.php?form=UserProfileEdit{@SID_ARG_2ND}"><img src="{icon}editS.png{/icon}" alt="" /> <span>{lang}wbb.header.userMenu.profile{/lang}</span></a></li>
 						{if MODULE_PM == 1 && $this->user->getPermission('user.pm.canUsePm')}
 							<li {if $this->user->pmUnreadCount} class="new"{/if} id="userMenuPm"><a href="index.php?page=PMList{@SID_ARG_2ND}"><img src="{icon}pm{if $this->user->pmUnreadCount}Full{else}Empty{/if}S.png{/icon}" alt="" /> <span>{lang}wbb.header.userMenu.pm{/lang}{if $this->user->pmUnreadCount} ({#$this->user->pmUnreadCount}){/if}</span></a>{if $this->user->pmTotalCount >= $this->user->getPermission('user.pm.maxPm')} <span class="pmBoxFull">{lang}wcf.pm.userMenu.mailboxIsFull{/lang}</span>{/if}</li>
